docs/move-overview.md

```diff
@@ -64,59 +64,31 @@ Now let us see how a programmer can interact with these modules and resources in
 // Simple peer-peer payment example.
 
 script {
-  // Use the LibraAccount module published on the blockchain at account address
-  // 0x0...1. 0x1 is shorthand that the language pads out to
-  // 16 bytes by adding leading zeroes.
-  use 0x1::LibraAccount;
-  // Use the LBR resource from the LBR module. Naming the resource here makes
-  // it possible to reference the resource without the module name.
-  use 0x1::LBR::LBR;
-  fun main(payer: &signer, payee: address, amount: u64) {
-    // Acquire the capability to withdraw from the payer's account.
-    let payer_withdrawal_cap = LibraAccount::extract_withdraw_capability(payer);
-    // Acquire a Libra<LBR> resource with value `amount` from the payer's
-    // account.  This will fail if the payer's balance is less than `amount`.
-    let coin = LibraAccount::withdraw_from<LBR>(&payer_withdrawal_cap, amount);
-    // Restore the capability back to the payer's account.
-    LibraAccount::restore_withdraw_capability(payer_withdrawal_cap);
-    // Move the `coin` resource into the account of `payee`. If there is no
-    // account at the address `payee`, this step will fail
-    // Note that we could have also written `deposit<LBR>` here, but the
-    // compiler is able to infer this for us based on the type of `coin`
-    LibraAccount::deposit(payer, payee, coin);
-  }
+// Use the LibraAccount module published on the blockchain at account address
+// 0x0...1. 0x1 is shorthand that the language pads out to
+// 16 bytes by adding leading zeroes.
+use 0x1::LibraAccount;
+// Use the LBR resource from the LBR module. Naming the resource here makes
+// it possible to reference the resource without the module name.
+use 0x1::LBR::LBR;
+
+fun peer_to_peer_lbr_payment(payer: &signer, payee: address, amount: u64) {
+  // Acquire the capability to withdraw from the payer's account.
+  let payer_withdrawal_cap = LibraAccount::extract_withdraw_capability(payer);
+  // Withdraw a value of `amount` LBR from the payer's account and deposit
+  // it into the account at the address `payee`. This will fail if the
+  // payer's balance is less than `amount` or if there is no account at the
+  // address `payee`. The `metadata` and `metadata_signature` arguments are
+  // not specified, so this could also fail if the transaction requires
+  // that information.
+  LibraAccount::pay_from<LBR>(&payer_withdrawal_cap, payee, amount, x"", x"");
```

Comment:
If `pay_from` is doing both the `withdraw` and `deposit`, then we will need to update an image we have for an update of the site where we show both of the latter being used, I think. I can show you when we next chat.

Reply:
Yes, the withdraw_from and deposit functions are now private to the LibraAccount module, and the pay_from function combines those two operations.